client/beadledom-client-guice/src/main/java/com/cerner/beadledom/client/BeadledomClientBuilderProvider.java

```diff
@@ -84,15 +84,19 @@ public BeadledomClientBuilder get() {
       }
     }
 
-    Injector injector = injectorProvider.get();
-    processInjector(injector, clientBuilder);
-    while (injector.getParent() != null) {
-      injector = injector.getParent();
-      processInjector(injector, clientBuilder);
+    Injector tempInjector = getInjector();
```

Comment:
I put it in a getter to avoid an issue of accidentally modifying the injector field in the while loop.

Ideally the injector field would be `final`, but cannot be since we must use method injection to initialize this class. Using a getter seemed like a decent alternative to avoid accidental modification.